src/ruuvitag_ble/parser.py

```diff
@@ -42,8 +42,7 @@ def _start_update(self, service_info: BluetoothServiceInfo) -> None:
         decoder = decoder_cls(raw_data)
 
         # Compute short identifier from MAC address
-        # (preferring the MAC address the tag broadcasts).
-        identifier = short_address(decoder.mac or service_info.address)
+        identifier = short_address(service_info.address)
```

Comment:
This looks like a spurious change?

Reply:
This goes with the removal of the mac explained above.  It seems cleaner to me to just use the service info and is one less thing to parse.  Unless you know of some reason the one extracted from the advertisement is preferred?